### DISCUSSION TITLE
New build release -regd

### DISCUSSION THREAD

**Discussion by KirubakaranSP** [apache/incubator-hugegraph#1663] [General]
I want to release a new build with latest changes.Can you please guide me how to create a build ?

**Comment by imbajin** [comment]
Excuse me, didn't see here before

And this problem has been solved in #1671 .

**Comment by imbajin** [comment]
@KirubakaranSP  Hi, due to long time with no response, could u please update or mark this conversation resolved? 

Thanks, and welcome to join apache community
